cmd/podman/common.go

```diff
@@ -221,6 +221,9 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"env", "e", []string{},
 		"Set environment variables in container",
 	)
+	createFlags.Bool(
+		"env-host", false, "Use all current host environment variables in container",
```

Comment:
Should this be `host-env` instead?

Reply:
I wanted to  keep all of the --env* flags the same, so they would be easier to discover.